src/dynamically-sized-types.md

```diff
@@ -20,7 +20,7 @@ types">DSTs</abbr>. Such types can only be used in certain cases:
   last field, this makes the struct itself a
   <abbr title="dynamically sized type">DST</abbr>.
 
-Notably: [variables], function parameters, [const] and [static] items must be
+***Note***: [variables], function parameters, [const] items, and [static] items must be
```

Comment:
Only need asterisks, and I forgot to mention the blockquote. Introduction.md is the source of truth here.

```suggestion
> **Note**: [variables], function parameters, [const] items, and [static] items must be
```